src/src/testruner/hillel/TestRunner.java

```diff
@@ -0,0 +1,66 @@
+package testruner.hillel;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.*;
+
+
+public class TestRunner {
+    private static final Logger logger = Logger.getLogger(TestRunner.class.getName());
+
+    public static void start(Class<?> testClass) {
+        if (testClass == null) {
+            throw new IllegalArgumentException("testClass cannot be null");
+        }
+
+        try {
+            Object testInstance = testClass.getDeclaredConstructor().newInstance();
+            Method[] methods = testClass.getDeclaredMethods();
+            Map<Integer, List<Method>> methodMap = new TreeMap<>();
+
+            Method beforeSuiteMethod = null;
+            Method afterSuiteMethod = null;
+
+            for (Method method : methods) {
+                if (method.isAnnotationPresent(Test.class)) {
+                    int priority = method.getAnnotation(Test.class).priority();
+                    methodMap.computeIfAbsent(priority, k -> new ArrayList<>()).add(method);
+                }
+                if (method.isAnnotationPresent(BeforeSuite.class)) {
+                    if (beforeSuiteMethod == null) {
+                        beforeSuiteMethod = method;
+                    } else {
+                        throw new RuntimeException("Multiple @BeforeSuite methods found in the Test Suite.");
```

Comment:
The message is not giving the user an explanation that only one method can be marked via `@BeforeSuite`